sdccc/src/main/java/com/draeger/medical/sdccc/tests/util/InitialMdibVersionPredicateWithUUID.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * This Source Code Form is subject to the terms of the MIT License.
+ * Copyright (c) 2023 Draegerwerk AG & Co. KGaA.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+package com.draeger.medical.sdccc.tests.util;
+
+import java.math.BigInteger;
+import java.util.function.Predicate;
+import org.apache.commons.lang3.tuple.Pair;
+import org.somda.sdc.biceps.model.message.AbstractReport;
+
+/**
+ * Predicate that does the same as InitialMdibVersionPredicate, but also passes the UUID Strings through.
+ */
+public class InitialMdibVersionPredicateWithUUID implements Predicate<Pair<AbstractReport, String>> {
+    private final BigInteger initialMdibVersion;
+    private boolean seenLargerVersion;
+
+    InitialMdibVersionPredicateWithUUID(final BigInteger initialValue) {
+        initialMdibVersion = initialValue;
+        seenLargerVersion = false;
+    }
+
+    @Override
+    public boolean test(final Pair<AbstractReport, String> pair) {
```

Comment:
The UUIDs seem to be unused.

Reply:
They are used in MdibHistorian.filterReportDuplicates to enrich the reason for Test Run Invalidation with References to the 2 Reports in question.

Reply:
I see, then the old InitialMdibVersionPredicate.class seems to be unused and can be removed.